lib/lua/raiseDelayedJobs.lua

```diff
@@ -10,13 +10,21 @@ returns number of jobs raised and the timestamp of the next job (within the near
 local now = tonumber(ARGV[1])
 
 -- raise any delayed jobs that are now valid by moving from delayed to waiting
-local raising = redis.call("zrangebyscore", KEYS[1], 0, ARGV[1])
-local numRaising = #raising
+local offset = 0
+-- There is a default 1024 element restriction
```

Comment:
is this a redis `lpush` limitation or a lua `unpack` limitation? can you comment here to clarify?

Reply:
nit: `s/There/there`

Reply:
I believed this limit is derived from the Lua config parameter, DEFAULT_STACK_SIZE (http://www.lua.org/source/4.0.1/llimits.h.html#DEFAULT_STACK_SIZE); however, upon deeper inspection, Redis uses Lua 5.1 and I can't find an equivalent parameter there.  It is possible this 1024 assertion is spurious; 
the closest equivalent I can find in Lua 5.1 is LUAI_MAXCSTACK and it's set to 8000 in Redis (http://download.redis.io/redis-stable/deps/lua/src/luaconf.h)